.env.example

```diff
@@ -12,3 +12,12 @@ PLATFORM_BACK_URL=http://localhost:8000
 OIDC_OP_ISSUER=https://iam-next.bimdata.io
 API_URL=https://api-next.bimdata.io
 APP_URL=http://localhost:8080
+
+# If launched on premises
+ON_PREMISE=True
```

Comment:
I think we could send our own mails through html and not online templates too. This should simplify the code. I also see a future i18n mail too and it could be hard to deal with many languages and many sending methods